### PR TITLE
updated compatible versions to future badge

### PR DESCRIPTION
**Pull request by elonmusksama**
HUGE CHANGES!!!!
updated the compatible versions to the future integrations badge

### CODE REVIEW

**Comment by dominikaaaa** [comment]
is it really considered compatibility if it tries to kill a zombie pigman with a steak while autowalking

**Comment by elonmusksama** [comment]
i mean baritone integration has nothing to do with futures kill aura but ok. iirc its cuz its detected as angry or possibly angry, 0x said that "not normal rotation movement for a pigman means they are like 80% sure angry or they just looked at you for some reason" and that there's no definitive was of detecting when they're hostile, in any case u can just disable hostile mobs in kill aura settings

**Comment by dominikaaaa** [comment]
> baritone integration has nothing to do with futures kill aura

Yea, because it isn't properly integrated

# :joy_cat:

**Comment by dominikaaaa** [comment]
> there's no definitive was of detecting when they're hostile

Holy shit I can't believe 0x said that lmao
image

**Comment by elonmusksama** [comment]
pigmen can be angry without running at you and eitherway thats not 100% reliable either

**Comment by dominikaaaa** [comment]
Idk how future fucked it up man but I've never had an issue with pigmen in non-future clients. This is hilarious :joy_cat:

**Comment by elonmusksama** [comment]
> Yea, because it isn't properly integrated
> # joy_cat
its is tho, but either way, this nitpicking is just personal beef between you and future so idk what this has to do with the actual baritone integration, it works completely fine and this badge was already added, meaning baritone has approved future as a client that successfully integrated baritone support. i just added the correct baritone versions to the badge lol

**Comment by elonmusksama** [comment]
> 
> 
> Idk how future fucked it up man but I've never had an issue with pigmen in non-future clients. This is hilarious joy_cat

im sure it doesnt attack pigmen in time, they get atleast 1 hit on you, but again idk how to code clients, im just here cuz i saw an outdated badge lol

**Comment by dominikaaaa** [comment]
Isn't this a duplicate then
image

**Comment by elonmusksama** [comment]
no, because john simply stated what version of future _initially_ added baritone integration (which is irrelevant seeing as the client constantly updates), while i added the versions of baritone that are compatible with futures integration (which are bound to change obviously, but are relevant to add in the readme)

**Comment by dominikaaaa** [comment]
^

**Comment by elonmusksama** [comment]
thank you bella, very cool!

**Comment by 0-x-2-2** [comment]
> is it really considered compatibility if it tries to kill a zombie pigman with a steak while autowalking

troll

this is legit has a setting to enable/disable.

**Comment by 0-x-2-2** [comment]
> > baritone integration has nothing to do with futures kill aura
> 
> Yea, because it isn't properly integrated
> 
> # 😹

how is the baritone integration with the killaura related to the entity relation system in future 🤔

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@
 [![Aristois add-on integration](https://img.shields.io/badge/Aristois%20add--on%20integration-v1.3.4%20/%20v1.4.1-green.svg)](https://gitlab.com/emc-mods-indrit/baritone_api)
 [![rootNET integration](https://img.shields.io/badge/rootNET%20integration-v1.2.11-green.svg)](https://rootnet.dev/)
 [![WWE integration](https://img.shields.io/badge/WWE%20%22integration%22-master%3F-green.svg)](https://wweclient.com/)
-[![Future integration](https://img.shields.io/badge/Future%20integration-Soon™%3F%3F%3F-red.svg)](https://futureclient.net/)
+[![Future integration](https://img.shields.io/badge/Future%20integration-v1.2.12%20%2F%20v1.3.6%20%2F%20v1.4.4-red)](https://futureclient.net/)
```
Comment (0-x-2-2) [inline README.md:30]: 1.2.10+ is officially supported but slightly older probably works but was never tested after more stuff was added to it.

**Pull request merged**